python/packages/autogen-ext/pyproject.toml

```diff
@@ -21,8 +21,9 @@ dependencies = [
 
 [project.optional-dependencies]
 langchain-tools = ["langchain_core~= 0.3.3"]
-azure-code-executor = ["azure-core"]
-docker-code-executor = ["docker~=7.0"]
+azure = ["azure-core", "azure-identity"]
+docker = ["docker~=7.0"]
+openai = ["openai>=1.3"]
```

Comment:
Please update the packages index page and relevant documentation pages if the extras are changing.

I think we should the old extras for a little time while the documentation is updated then we can drop them later

Reply:
Done. Hadn't been aware of those.

 BTW, the change here is to simplify the extras logic since we removed the namespace packages. Before we could possibly have `autogen_ext.models.azure` and `autogen_ext.code_executor.azure` and I thought we may want to have specific extras for each namespace package but since the idea was abandoned I thought it would be worth it to simplify the extras as well. Let me know if this is not aligned with the design.

Reply:
I think simplified extras that are easy to understand is for the best, sounds good to me!